packages/runtime-core/__tests__/apiOptions.spec.ts

```diff
@@ -512,15 +546,60 @@ describe('api: options', () => {
       mounted() {
         calls.push('comp')
       },
-      render(this: any) {
+      render() {
         return `${this.a}${this.b}`
       }
-    }
+    })
 
     expect(renderToString(h(Comp))).toBe(`12`)
     expect(calls).toEqual(['base', 'comp'])
   })
 
+  test('extends with mixins', () => {
```

Comment:
Same as the previous component, we probably want to test the access of the component who's extending.